examples/echo.rs

```diff
@@ -176,6 +180,7 @@ fn main() {
         Box::new(friend_connections.run().map_err(Error::from)),
         Box::new(lossless_future),
         Box::new(lossy_future),
+        Box::new(friend_future)
```

Comment:
It makes sense to put trailing comma here so that if we need to add one more future we will change only one line - that allows to have more precise git blames.